clippy_lints/src/non_expressive_names.rs

```diff
@@ -133,6 +153,15 @@ impl<'a, 'tcx, 'b> SimilarNamesNameVisitor<'a, 'tcx, 'b> {
         if interned_name.chars().any(char::is_uppercase) {
             return;
         }
+        if interned_name.chars().all(|c| c.is_digit(10) || c == '_') {
+            span_lint(
+                self.0.cx,
+                JUST_UNDERSCORES_AND_NUMBERS,
+                span,
+                "binding whose name is just underscores and digits",
```

Comment:
add some suggestion on what to do, too. Maybe like the other message "consider choosing a more descriptive name"